### DISCUSSION TITLE
Does this solve the multiple launcher installed issue?

### DISCUSSION THREAD

**Discussion by fathonyfath** [moraroy/NonSteamLaunchers-On-Steam-Deck#236] [Q&A]
Hi, I was wondering if this solution solves the issue where different games from the same launcher install launcher to each `wineprefix` of the game?

For example, what I understand is if I install RDR2 and GTA games, I will have different Rockstar launchers installed on each of the `wineprefix`. If I set the `STEAM_COMPAT_DATA_PATH` to all the rockstar games to the prefix where NonSteamLaunchers installs Rockstar launcher, would that solve the issue?

**Comment by moraroy** [comment]
Yes it does
